src/bdev.rs

```diff
@@ -110,7 +112,27 @@ where
         }
     }
 
+    /// Returns claim module raw pointer.
+    fn claim_module_ptr(&self) -> *mut spdk_bdev_module {
```

Comment:
```suggestion
    /// Returns claim module raw pointer.
    /// TODO: support multiple claims
    fn first_claim_module_ptr(&self) -> *mut spdk_bdev_module {
```

Maybe this would make it clearer?

Reply:
ok, will do. I'll also mark it as deprecated.